docs/uCredit/frontend/components/majors.md

```diff
@@ -0,0 +1,1494 @@
+
+## Layout of a `FineReq`
+
+Each `FineReq` object is a custom JavaScript Object with its own fields also defined in `commonTypes.tsx`. The definition of a `FineReq` object is shown below and is similar to the definition of a `DistributionObj`. Once again, the `?` after specific member fields indicates that these member fields are optional and **NOT** required when creating a new `FineReq`.
+
+```
+export type FineReq = {
+    required_credits: number;
+    description: string;
+    criteria: string;
+    exclusive?: boolean;
+};
+```
+
+Each `FineReq` contains the following member fields:
+
+| Field                  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `required_credits`     | Number representing the total number of credits required to satisfy the fine requirement.                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `description`          | String representing a brief description of the fine requirement.                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Is it possible to mention how the descriptions are formatted for fine reqs? For example,

```
''<b>General Physics I</b> <br /> AS.171.101 General Physics: Physical Science Majors I <br /> <i>OR</i> <br /> AS.171.107 General Physics for Physical Sciences Majors I (AL).''
```

Is there a structure to the ``<b>``, ``<br>``, and ``<i>`` tags?

Reply:
Just added a line to specify that HTML tags are supported as long as they are all closed properly.

Reply:
Assuming this convo is resolved.